uint16 header
string id
uint32 int_config_val
float32 float_config_val
